internal/ls/autoimports.go

```diff
@@ -91,7 +91,9 @@ func (e *exportInfoMap) get(importingFile tspath.Path, ch *checker.Checker, key
 	if e.usableByFileName != importingFile {
 		return nil
 	}
-	return core.Map(e.exportInfo.Get(key), func(info CachedSymbolExportInfo) *SymbolExportInfo { return e.rehydrateCachedInfo(ch, info) })
+	return core.MapFiltered(e.exportInfo.Get(key), func(info CachedSymbolExportInfo) (*SymbolExportInfo, bool) {
```

Comment:
Just make these `MapNonNil` - every `false` condition is also `nil` anyway.

I don't know if `tryRehydrateCachedInfo` would be a better name to indicate that it might return `nil` now.